app/jobs/metais/sync_codelists_job.rb

```diff
@@ -0,0 +1,7 @@
+class Metais::SyncCodelistsJob < ApplicationJob
+  queue_as :metais
+
+  def perform()
+    # TODO
```

Comment:
Tu asi nieco chyba.

Reply:
@jsuchal no to sú tie číselníky. Je ich tam rádovo 6. Nie je problém spraviť pre každý jednu tabuľku a naplniť ich. Pri ITMS som ale videl, že sú tabuľky `ciselniky` a `hodnoty_ciselnikov`. Mám to teda spraviť rovnako pri tomto metais?

Reply:
Ciselniky dajme normálne slušne tabuľky.  Ak ich je 6 tak to dáme.